feature/policy_forwarding/vrf_selection/otg_tests/multiple_vrfs_and_gue_decap/README.md

```diff
@@ -0,0 +1,403 @@
+                }
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "policy-definitions": {
+      "policy-definition": [
+        {
+          "config": {
+            "name": "BGP_EXPORT_B2_VRF"
+          },
+          "name": "BGP_EXPORT_B2_VRF",
+          "statements": {
+            "statement": [
+              {
+                "actions": {
+                  "config": {
+                    "policy-result": "ACCEPT_ROUTE"
+                  }
+                },
+                "conditions": {
+                  "match-prefix-set": {
+                    "config": {
+                      "prefix-set": "IPv4Prefixes_IBGP"
+                    }
+                  }
+                },
+                "config": {
+                  "name": "IPv4Prefix_IBGP_EXPORT"
+                },
+                "name": "IPv4Prefix_IBGP_EXPORT"
+              },
+              {
+                "conditions": {
+                  "match-prefix-set": {
+                    "config": {
+                      "prefix-set": "IPv6Prefixes_IBGP"
+                    }
+                  }
+                },
+                "config": {
+                  "name": "IPv6Prefix_IBGP_EXPORT"
+                },
+                "name": "IPv6Prefix_IBGP_EXPORT"
+              }
+            ]
+          }
+        }
+      ]
+    }
+  }
+}
+```
+
+## OpenConfig Path and RPC Coverage
+
+```yaml
+paths:
+
+/network-instances/network-instance/policy-forwarding/policies/policy/config/policy-id:
+/network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/config/ipv4/config/destination-address-prefix-set:
+/network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/config/ipv4/config/protocol:
+/network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/transport/config/destination-port:
+/network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/action/decapsulate-gue:
+/network-instances/network-instance/inter-instance-policies/apply-policy/config/export-policy:
+/network-instances/network-instance/inter-instance-policies/apply-policy/config/import-policy:
+/routing-policy/policy-definitions/policy-definition/config/name:
+/routing-policy/policy-definitions/policy-definition/statements/statement/config/name:
+/routing-policy/policy-definitions/policy-definition/statements/statement/name/actions/policy-result:
+/routing-policy/policy-definitions/policy-definition/statements/conditions/oc-bgp-pol:bgp-conditions/oc-bgp-pol:match-community-set:
+/routing-policy/policy-definitions/policy-definition/statements/statement/actions/config/policy-result:
+/routing-policy/policy-definitions/policy-definition/statements/statement/conditions/match-prefix-set/prefix-set:
+/routing-policy/defined-sets/prefix-sets/prefix-set/prefixes/prefix/masklength-range:
+/routing-policy/defined-sets/prefix-sets/prefix-set/prefixes/prefix/ip-prefix:
```

Comment:
Paths need to be leaves, so they should have a "config" or "state" somewhere in them.

```suggestion
/routing-policy/policy-definitions/policy-definition/statements/statement/conditions/match-prefix-set/config/prefix-set:
/routing-policy/defined-sets/prefix-sets/prefix-set/prefixes/prefix/config/masklength-range:
/routing-policy/defined-sets/prefix-sets/prefix-set/prefixes/prefix/config/ip-prefix:
```